### ISSUE TITLE
ARIA 1.1 pattern doesn't require "combobox" to have aria-controls attribute

### ISSUE THREAD

**Issue by KuSh** [A11yance/aria-query#43]
Hi,

I'm trying to implement accessible component following w3c recommendation and examples and the following line cause jsx-a11y to output an error :
https://github.com/A11yance/aria-query/blob/f91d8187a0ff7f8a60892ed86e8551c4281e08d2/scripts/roles.json#L432

According to https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html in ARIA 1.1 pattern the required attribute is aria-owns and the aria-controls attribute should be on the child textbox

regards,
Nicolas

**Comment by eps1lon** [comment]
`aria-controls` is currently required in the [latest editors draft of aria](https://w3c.github.io/aria/#combobox). 

My guess is that aria-query targets the latest editors drafts. This is fairly common for these kind of libraries as far as I know (jsdom follows editors drafts as well).

**Comment by KuSh** [comment]
Ow ... I'll revert to Aria 1.0 pattern then 😭
Thanks for your feedback 👍

**Comment by eps1lon** [comment]
> Ow ... I'll revert to Aria 1.0 pattern then 

Your linter shouldn't dictate the UX  you ship. The aria 1.2 pattern is hopefully better than 1.1 and 1.1. better than 1.0. Please rather ignore the lint error at these lines instead of changing your component.

**Comment by KuSh** [comment]
I'm generally ok with you but since for that particular pattern 1.2 seems compatible with 1.0 and 1.2 draft is insinuating that there are problems correctly implementing 1.1 pattern I think I'll downgrade that one.

**Comment by KuSh** [comment]
Thanks a lot for your feedback ! Much appreciated

**Comment by mcking65** [comment]
Yes, please use the 1.2 pattern. Basically, it is the same as 1.0 except that you replace aria-owns with aria-controls. If you want some background on this see:
https://github.com/w3c/aria/wiki/Resolving-ARIA-1.1-Combobox-Issues

**Comment by KuSh** [comment]
> If you want some background on this see:
> https://github.com/w3c/aria/wiki/Resolving-ARIA-1.1-Combobox-Issues
Thanks that's very informative !

**Comment by jessebeach** [comment]
#46 addresses the ARIA 1.2 spec update. A release of this module is forthcoming in the next week.

@KuSh thank you for the Issue!
